js/app.js

```diff
@@ -0,0 +1,218 @@
+function ProjectContructor(image, name, company, description, technology, buttonId) {
+  this.image = image;
+  this.name = name;
+  this.company = company;
+  this.description = description;
+  this.technology = technology;
+  this.buttonId = buttonId;
+  this.buttonId += '-button';
+}
+
+const tonic = new ProjectContructor(
+  'media/Snapshoot Portfolio desk.png',
+  'Tonic',
+  {
+    name: 'CANOPY',
+    position: 'Back End Dev',
+    date: '2015',
+  },
+  'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  ['HTML', 'CSS', 'JavaScript'],
+  'tonic',
+);
+
+const multiPostSories = new ProjectContructor(
+  'media/Snapshoot Portfolio desk 1.png',
+  'Multi-Post Stories',
+  { name: 'FACEBOOK', position: 'Full Stack Dev', date: '2015' },
+  'Experimental content creation feature that allows users to <br> add to an existing story over the course of a day without spamming their friends.',
+  ['HTML', 'Ruby on rails', 'CSS', 'Javascript'],
+  'multi-post-stories',
+);
+
```

Comment:
Please note that your desktop popup is a bit off from the original design. Kindly adjust the design a bit for a better UI 👍🏽
Figma Design | Your Design
![Screenshot from 2023-07-26 18-35-05](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/44624138/8328125e-3c9b-4488-9e1b-3cc484c06281) | ![Screenshot from 2023-07-26 18-39-47](https://github.com/maximoortelli/Portfolio-setup-and-mobile--first/assets/44624138/57d4cfdf-ac80-4a7e-a059-96fe8455aca6)

Reply:
Please note that the icon on the `see source ` button is currently broken, kindly fix it. 😉

Reply:
Done👍